bin/hbase-daemon.sh

```diff
@@ -155,12 +155,20 @@ JAVA=$JAVA_HOME/bin/java
 export HBASE_LOG_PREFIX=hbase-$HBASE_IDENT_STRING-$command-$HOSTNAME
 export HBASE_LOGFILE=$HBASE_LOG_PREFIX.log
 
-if [ -z "${HBASE_ROOT_LOGGER}" ]; then
-export HBASE_ROOT_LOGGER=${HBASE_ROOT_LOGGER:-"INFO,RFA"}
+if [ -z "${HBASE_ROOT_LOGGER_LEVEL}" ]; then
+export HBASE_ROOT_LOGGER_LEVEL=${HBASE_ROOT_LOGGER_LEVEL:-"INFO"}
 fi
 
-if [ -z "${HBASE_SECURITY_LOGGER}" ]; then
```

Comment:
If provided this string we could munge this string into LOGGER_APPENDER and LOGGER_LEVEL and warn about the change in environment variable usage for additional compatibility.

Reply:
Ditto.